### PR TITLE
Trigger and wait for graph validation

### PR DESCRIPTION
**Pull request by jacobwindsor**
dcp-515

Adds testing for graph validation state transitions

### CODE REVIEW

File: tests/runners/update_submission_runner.py [hunk 1]
```diff
@@ -103,6 +105,9 @@ def run_primary_submission(self, dataset_name):
         # Disable indexing since this is an internal test for ingest, we don't need to trigger analysis pipelines
         submission_manager.submission_envelope.disable_indexing()
 
+        submission_manager.graph_validate_envelope()
+        submission_manager.wait_for_envelope_to_be_graph_valid()
```
Comment (aaclan-ebi) [inline tests/runners/update_submission_runner.py]: could rename the methods to
`validate_envelope_graph`or `validate_graph`
 `wait_for_envelope_to_have_valid_graph` for readability

**Review by aaclan-ebi — DISMISSED** [review]

File: tests/runners/update_submission_runner.py (hunk 1 above)
Comment (jacobwindsor) [reply tests/runners/update_submission_runner.py]: Done!

**Review by aaclan-ebi — DISMISSED** [review]

**Review by ke4 — DISMISSED** [review]
LGTM

**Review by aaclan-ebi — APPROVED** [review]

**Review by ke4 — APPROVED** [review]
Good refactor! Simplified -> cleaner code! 👍

**Pull request merged**